### ISSUE TITLE
Feature Request: Enhance fritzboxtr064.cfg

### ISSUE THREAD

**Issue by bestfan** [gitbock/fritzboxtr064#42]
I was wondering if someone has already thought to apply this binding for other routers with
TR-064 support? The Zyxel VMG1312-B30A would be a candidate. Different port setting (49431 instead of 49000 at the Fritzbox) can be adjusted in the config file. Unfortunately, adjustment of the description .xml file (devicedesc.xml instead of tr064desc.xml at the Fritxbox) is not possible (hard coded in "Tr064Comm.java"). If someone already came around this topic I am happy to test and share my experience.

**Comment by rbm78bln** [comment]
Yes, currently fiddeling around with the same device...yet I am at the very beginning...
